node/utils/attributes.ts

```diff
@@ -142,48 +139,30 @@ const convertValues = (
     }
   }
 
-  // FIXME: Remove this when creating the specificationType for filters
-  if (
-    attribute.type === 'number' &&
-    attribute.key === 'quilometragem' &&
-    account === 'localizaseminovos'
-  ) {
-    return {
-      type: 'NUMBER',
-      values: attribute.values.map((value: any) => {
-        return {
-          quantity: value.count,
-          name: unescape(value.label),
-          key: attribute.key,
-          value: value.key,
-          selected: value.active,
-          range: {
-            from: parseFloat(
-              isNaN(value.from) ? attribute.minValue : value.from
-            ),
-            to: parseFloat(isNaN(value.to) ? attribute.maxValue : value.to),
-          },
-        }
-      }),
-    }
-  }
-
   // For other `number` and `location` attributes we use TEXT based filters (checkboxes),
   // with buckets (e.g.: 0 - 30, 30 - 90, etc).
   if (attribute.type === 'number' || attribute.type === 'location') {
     const valuePrefix = attribute.type === 'location' ? 'l:' : ''
 
     // When a bucket is active, we should only show it, and none of the othter options.
     if (attribute.active) {
+      const from = !isNaN(parseFloat(attribute.activeFrom ?? '')) ? parseFloat(attribute.activeFrom!) : attribute.minValue
```

Comment:
can this value be `null` at any time? if not, this function could be simpler like:
```suggestion
      const from = parseFloat(isNaN(attribute.activeFrom) ? attribute.minValue : attribute.activeFrom)
```

Reply:
Yes, it can be `null`, but not when it's active. If there was type inference based on a boolean value like string ones, we could have `ActiveNumericalAttribute` and `InactiveNumericalAttributes` that have different fields, but I guess the way it's right now is fine, without adding too much overhead, I'll resolve this discussion, ok? 😄 The other problem is that `activeFrom` and `activeTo` are strings that can be `'*'`, while `minValue` and `maxValue` so using them on `parseFloat` is a type error to TS. 🤷‍♂️   

![image](https://user-images.githubusercontent.com/34144667/98250448-31ea0180-1f56-11eb-99d6-a8512dfea10d.png)